package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "node.js_course_practice",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "start": "nodemon ./src/index.ts",
+    "start:dev": "npx nodemon",
+    "build": "rimraf ./build && tsc",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ivan-kuhta/Node.js_Course_Practice.git"
+  },
+  "author": "Ivan Kukhta",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ivan-kuhta/Node.js_Course_Practice/issues"
+  },
+  "homepage": "https://github.com/ivan-kuhta/Node.js_Course_Practice#readme",
+  "dependencies": {
+    "express": "^4.18.2",
+    "nodemon": "^3.0.1",
+    "swagger-jsdoc": "^6.2.8",
+    "swagger-ui": "^5.9.0",
+    "swagger-ui-express": "^5.0.0"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.20",
+    "@types/mongoose": "^5.11.97",
+    "@types/node": "^20.8.9",
+    "@types/swagger-jsdoc": "^6.0.2",
+    "@types/swagger-ui-express": "^4.1.5",
+    "mongodb": "^6.2.0",
+    "mongoose": "^8.0.0",
```

Comment:
Why "mongoose" package is in devDependencies?
Please, also add eslint

Reply:
It really makes sense. I will also add eslint.